service.go

```diff
@@ -274,6 +274,23 @@ func (svc *Service) HandleEvent(ctx context.Context, event *nostr.Event) (result
 		return nil, nil
 	}
 
+	if svc.lnClient == nil {
+		svc.Logger.WithFields(logrus.Fields{
+			"nostrPubkey": event.PubKey,
+		}).Errorf("Received an event before ln client is initialized")
+
+		ss, err := nip04.ComputeSharedSecret(event.PubKey, svc.cfg.NostrSecretKey)
+		if err != nil {
+			return nil, err
+		}
+		return svc.createResponse(event, Nip47Response{
+			Error: &Nip47Error{
+				Code:    NIP_47_ERROR_INTERNAL,
```

Comment:
I do not think `INTERNAL` is the best code here because it's more that the user has not setup their connection yet.

What about `RESTRICTED`? or `OTHER`?

Reply:
Update: this doesn't make sense